librz/analysis/dwarf_process.c

```diff
@@ -48,10 +48,10 @@ typedef struct dwarf_var_location_t {
 } VariableLocation;
 
 typedef enum variable_kind_t {
-	INVALID = 0,
-	FORMAL_PARAMETER,
-	VARIABLE,
-	UNSPECIFIED_PARAMETERS,
+	INVALID = RZ_ANALYSIS_VAR_KIND_INVALID,
+	FORMAL_PARAMETER = RZ_ANALYSIS_VAR_KIND_FORMAL_PARAMETER,
+	VARIABLE = RZ_ANALYSIS_VAR_KIND_VARIABLE,
+	UNSPECIFIED_PARAMETERS = RZ_ANALYSIS_VAR_KIND_UNSPECIFIED_PARAMETERS,
 } VariableKind;
```

Comment:
If these are the same, why not use directly the same enum?

Reply:
You are right.